google/cloud/security/scanner/scanners/fw_rules_scanner.py

```diff
@@ -23,6 +23,7 @@
 
 from google.cloud.security.common.data_access import csv_writer
 from google.cloud.security.common.data_access import firewall_rule_dao
+from google.cloud.security.common.gcp_type.resource import ResourceType
```

Comment:
I think our general pattern for using modules is to import the module (not the classname), then use

module.ClassName(...)

so you'd have 

`from google.cloud.security.common.gcp_type import resource_type`

and on line 191, you'd have:

```
resource_type.ResourceType.FIREWALL_RULE: ...
```